source/client/output_formatter_impl.cc

```diff
@@ -187,6 +188,7 @@ std::string FortioOutputFormatterImpl::formatProto(const nighthawk::client::Outp
 
   // TODO(#182): Not needed but nice to have, displays in the UI
   fortio_output.set_labels("");
+  fortio_output.set_version("0.0");
```

Comment:
Don't we want to set this to the actual version?

Reply:
> Don't we want to set this to the actual version?

I'm sorry, I had another look at this, and I somehow misunderstood your comment here.
That is correct, and this is fixed now.